tests/math/test_is_independent.py

```diff
@@ -519,7 +519,7 @@ def dependent_circuit(x, y, z):
 
         dependent_functions = [
             dependent_circuit,
-            torch.tensor,
+            torch.as_tensor,
```

Comment:
Applying `torch.tensor` to a Torch tensor raises a warning to use an appropriate copying method. We here just want to test some form of identity function, so `as_tensor` is as good as `tensor` and it does not raise the warning :)